v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -3,72 +3,124 @@ import {autorun} from "mobx"
 import {drag, select} from "d3"
 import {useAxisLayoutContext} from "../../axis/models/axis-layout-context"
 import {ScaleNumericBaseType} from "../../axis/axis-types"
-import {kGraphClassSelector} from "../graphing-types"
+import {kGraphAdornmentsClassSelector} from "../graphing-types"
 import {INumericAxisModel} from "../../axis/models/axis-model"
 import {equationString, IAxisIntercepts, lineToAxisIntercepts} from "../utilities/graph-utils"
 import {IMovableLineModel} from "./adornment-models"
+import { useDataConfigurationContext } from "../hooks/use-data-configuration-context"
 import "./movable-line.scss"
 
 export const MovableLine = (props: {
+  lineKey?: string
   model: IMovableLineModel
+  plotHeight: number
+  plotIndex: number
+  plotWidth: number
+  transform?: string
   xAxis: INumericAxisModel
   yAxis: INumericAxisModel
-  transform: string
 }) => {
-  const {model, xAxis, yAxis, transform} = props,
+  const {lineKey='', model, plotHeight, plotIndex, plotWidth, transform, xAxis, yAxis} = props,
+    data = useDataConfigurationContext(),
```

Comment:
We frequently use `data` to refer to the `DataSet`; `dataConfig` would be more conventional.